utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -1016,4 +1017,23 @@ inline void compare_output(T* output, string funcName, RpptDescPtr srcDescPtr, R
         qaResults << status << std::endl;
         qaResults.close();
     }
+}
+
+inline void randomize(unsigned int arr[], unsigned int n)
```

Comment:
Pls add one liner comment above this function stating what it takes in and what it gives, or its use case